packages/components/NativeRadioButton/macos/RadioButtonViewManager.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+@objc(RadioButtonViewManager)
+class RadioButtonViewManager: RCTViewManager {
+
+	override func view()->NSView! {
+		let radioButton = RadioButton.init(radioButtonWithTitle: "", target: nil, action: nil)
+		radioButton.action = #selector(radioButton.sendCallback)
+		radioButton.target = radioButton
+		return radioButton
+	}
```

Comment:
If you override init in your subclass (I think you have to override the designated initializer), you could move all this logic to inside RadioButton.swift, and just do 
`return RadioButton()`. You can see me do a similar thing in MenuButton.swift.

Reply:
Yeah I tried that and it didn't work. The problem is init.radioButtonWithTitle is a convenience initializer, I don't think swift allows a call to a convenience initializer inside a designated initializer.